src/components/Reports/GoalCalculator/SummaryReport/MpdGoal/MpdGoalTable.tsx

```diff
@@ -52,105 +59,106 @@ export const MpdGoalTable: React.FC<MpdGoalTableProps> = ({ goal }) => {
     [locale],
   );
 
-  const ministryExpenses = goal.ministryExpenses;
+  const goalCalculation = goalCalculationResult.data?.goalCalculation;
   const rows = useMemo((): MpdGoalRow[] => {
-    const ministryExpenseRows: MpdGoalRow[] = [
-      ...ministryExpenses.primaryCategories.map((category, index) => {
-        const lineNumber = String.fromCharCode(65 + index);
-        return {
-          line: `3${lineNumber}`,
-          category: t(category.label),
-          amount: category.amount,
-          reference: Math.random() * 391 + 10,
-        };
-      }),
-    ];
+    const ministryExpenseRows: MpdGoalRow[] =
+      goalCalculation?.ministryFamily.primaryBudgetCategories.map(
+        (category, index) => {
+          const lineNumber = String.fromCharCode(65 + index);
+          return {
+            line: `3${lineNumber}`,
+            category: t(category.label),
+            amount: calculateCategoryTotal(category),
+            reference: Math.random() * 391 + 10,
```

Comment:
Somehow, we're missing 2 expenses.

Reply:
https://docs.google.com/document/d/1Rj4Zz_ITF0cFIvpTqdybfLjXdHajENPTOlUX3hnomIg/edit?tab=t.0#heading=h.8c1acqqnggaq

Reply:
But something is off with the calculations.

<img width="612" height="428" alt="Image" src="https://github.com/user-attachments/assets/87fa8587-df08-46db-b976-f7c8d749e516" />

Reply:
sheet for reference: https://docs.google.com/spreadsheets/d/1yE3XRQudynyYE6LLT9zBFvTm6xjaQbHIOc2c5qqV7cU/edit?gid=1325063116#gid=1325063116